pkg/capture/capture_manager.go

```diff
@@ -580,12 +583,12 @@ func (cm *Manager) performWriteout(ctx context.Context, timestamp time.Time, ifa
 	writeoutChan := make(chan capturetypes.TaggedAggFlowMap, writeout.WriteoutsChanDepth)
 	doneChan := cm.writeoutHandler.HandleWriteout(ctx, timestamp, writeoutChan)
 
+	cm.Lock()
```

Comment:
This was also incorrect. The global lock must encompass the whole (unprotected) rotation, otherwise it's possible that a scheduled one runs in parallel to one triggered from an interface down. TBH this is the most likely candidate for the deadlocks (because we've seen it happen in exactly these scenarios quite frequently - interface changes + scheduled rotation)....